Examples/java/src/test/java/software/amazon/cryptography/example/downstream/ESDKTest.java

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.amazon.cryptography.example.downstream;
+
+import static software.amazon.cryptography.example.Fixtures.HV2_BRANCH_KEY_ID;
+import static software.amazon.cryptography.example.Fixtures.KEYSTORE_KMS_ARN;
+
+import com.amazonaws.encryptionsdk.AwsCrypto;
+import com.amazonaws.encryptionsdk.CryptoResult;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import software.amazon.cryptography.example.hierarchy.KeyStoreProvider;
+import software.amazon.cryptography.keystore.KeyStore;
+import software.amazon.cryptography.materialproviders.IKeyring;
+import software.amazon.cryptography.materialproviders.MaterialProviders;
+import software.amazon.cryptography.materialproviders.model.CacheType;
+import software.amazon.cryptography.materialproviders.model.CreateAwsKmsHierarchicalKeyringInput;
+import software.amazon.cryptography.materialproviders.model.MaterialProvidersConfig;
+import software.amazon.cryptography.materialproviders.model.NoCache;
+
+public class ESDKTest {
```

Comment:
This is a cool test, but
* Should this be in the examples test folder?
* I don't think it's ideal to write testing code with the ESDK inside the MPL. I thought the ideal state for downstream testing is that the MPL's CI would pull the ESDK from Maven, then run the ESDK's tests from the MPL's CI. That gets us better test coverage from the MPL and only one set of tests. But I know that is work, so this seems fine for now. But if that is the ideal state, can you add a TODO to change that?

Reply:
The ESDK's tests are not in Maven... 
so it would be impossible to to the ESDK's tests without significant refactoring.

About a TODO: this branch is for HV-2. The SIM/Story I am assigned is HV-2.
I think MPL Downstream Testing is worthy SIM/Story/work,
and this is related to it (maybe as a POC),
but MPL Downstream Testing should still,
in my opinion, 
be considered as un-started/un-funded.

Hence, I am against adding a TODO... 
I like TODOs for things I am "funded" to do.